connectors/filtering/basic_rule.py

```diff
@@ -114,13 +117,22 @@ def __eq__(self, other):
 
 
 class BasicRuleEngine:
+    """Class executing a list of basic rules in order against a document."""
```

Comment:
Same as above - comment can explain better:

1. What does it actually do - current description is so abstract it does not explain what the class is supposed to be used for
2. Provide an example of usage